classes/class-p4-post-campaign.php

```diff
@@ -352,34 +317,22 @@ private static function campaign_field(
 		 * @return array The values that will be used for the css variables.
 		 */
 		public static function css_vars( array $meta ): array {
-			$theme = $meta['theme'] ?? $meta['_campaign_page_template'];
-
 			// Set specific CSS for Montserrat.
 			$special_weight_fonts = [
 				'Montserrat'       => '900',
 				'Montserrat_Light' => '500',
 			];
 
-			$header_primary_font =
-				'Montserrat_Light' === $meta['campaign_header_primary'] ?? null
+			$header_primary_font = 'Montserrat_Light' === ( $meta['campaign_header_primary'] ?? null )
 					? 'Montserrat'
 					: $meta['campaign_header_primary'] ?? null;
 
-			$campaigns_font_map = [
-				'default'   => 'lora',
-				'antarctic' => 'sanctuary',
-				'arctic'    => 'Save the Arctic',
-				'climate'   => 'Jost',
-				'forest'    => 'Kanit',
-				'oceans'    => 'Montserrat',
-				'oil'       => 'Anton',
-				'plastic'   => 'Montserrat',
-			];
-
-			$campaign_font = $campaigns_font_map[ $theme ?: 'default' ];
-
-			if ( 'campaign' === $meta['campaign_body_font'] ) {
-				$body_font = $campaign_font;
+			if (
```

Comment:
The campaign fonts map is something I didn't take over from the original campaigns code initially. I had created it so that the setting's value is always used. However all those values come from the css files and there is already a rule there which sets them, so it's a bit silly what I did there and we can just only apply the rule if anything but 'campaign' is chosen.